app/src/main/java/org/fossasia/openevent/app/data/event/Event.java

```diff
@@ -105,12 +107,14 @@
     @JsonSerialize(using = ObservableStringSerializer.class)
     @JsonDeserialize(using = ObservableStringDeserializer.class)
     public ObservableString endsAt = new ObservableString();
+    @ColumnIgnore
+    @JsonIgnore
+    public ObservableBoolean selected = new ObservableBoolean();
```

Comment:
@sridharjajoo As you can see in other PR's, we are avoiding the use of ObservableBoolean. So please use Boolean instead.

Reply:
I tried changing Boolean from ObservableBoolean but the thing is that the highlighting in blue color which was happening earlier isn't happening now.

Reply:
You can use ObservableBoolean, but out of the class and in the form of a Map inside presenter

Reply:
@iamareebjamal Done the changes